public/index.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+<link>
     <meta charset="UTF-8">
+    <meta name="keywords" content="HTML, CSS, JavaScript, React, ReactJS, Shopping, Shopping App, Grocery, Grocery Shopping">
```

Comment:
put relevant, remove coding etc. put like brands etc. for SEO

Reply:
Added some